pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -37,13 +40,33 @@ def _target_satellite_host(request, satellite_factory):
         yield
 
 
+@lru_cache
+def capsule_cdn_register(hostname=None):
+    cap = Capsule(hostname=hostname)
+    # Enabling CDN repos on Capsule Host
+    cap.register_to_cdn()
+    # Enabling OHSnap repos on Capsule host
```

Comment:
Could you change this statement so it matches with what is being enabled? If you look closely, none of these repositories come from Ohsnap.
https://github.com/SatelliteQE/robottelo/blob/7bdd3207149d9c7a120adae177310d8f8a086aa8/robottelo/constants/__init__.py#L254-L264

Reply:
Yup! Updated that during TFA in 6.14 branch https://github.com/jyejare/robottelo/tree/614_Capsule_n_minus, even now I pushed it here!